Formula/f/ffmpeg.rb

```diff
@@ -96,6 +96,9 @@ class Ffmpeg < Formula
   end
 
   def install
+    # The new linker leads to duplicate symbol issue https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues/140
+    ENV.append "LDFLAGS", "-Wl,-ld_classic" if DevelopmentTools.clang_build_version >= 1500
+
```

Comment:
I think we'll be handling this in brew, @Bo98 and @fxcoudert probably have an opnion

Reply:
I think passing this for every formula in brew is too invasive and hard to reverse. It's better for now to do it only in the few formulas that need it (at this stage, out of 2400 already bottled: GCC, HDF5, OpenMPI and Node).